routes/admin.js

```diff
@@ -2,12 +2,16 @@ const express = require('express');
 
 const validationController = require('../controllers/validationController')
 const indexController = require('../controllers/playersIndexController');
+const eventsIndexController = require('../controllers/eventsIndexController');
 
 const router = express.Router();
 
 // Get all players info
 // /admin/players => GET
 // validate a password to get players infos
 router.get('/players/', validationController.validateAdminPassword, indexController.getIndex);
+// Get all events info
```

Comment:
I think `/admin` is not a good place for `events` endpoint. Because this is not only for Admin.
Just `/events` would be better.

Reply:
For some reason, I can not the GET request......

![Screenshot 2021-10-18 at 11 13 35](https://user-images.githubusercontent.com/60405643/137702975-ac117e1f-9937-45be-8281-0da69ef2ef79.png)

Reply:
Look like you're testing the production. But probably the change is not deployed on Production? 
Maybe you want to use `localhost:7000`?